### ISSUE TITLE
Data not resolved correctly

### ISSUE THREAD

**Issue by nemonemi** [stargaze-co/react-router-typesafe#3]
I have a route that has a nested route.
They both have loaders, and when I try to access the parent route data, I get the child loader data, regardless of the ID that I pass.

```ts
    {
      path: '/', // Base URL, here we serve the application. All the other URLs are nested underneath and define their own paths that append to the base URL
      loader: currentUserLoader,
      element: <App />,
      id: 'currentUser',
      children: [
        {
          loader: qualificationsLoader,
          path: `/qualifications`,
...
}
```
```ts
const currentUser = useRouteLoaderData<typeof currentUserLoader>('currentUser');
// Types are correct, but it returns qualifications, not the currentUser
```

**Comment by fredericoo** [comment]
I clearly need some code review. It’s a typo on re-exporting the original function. On it ◡̈

**Comment by fredericoo** [comment]
Patched on `1.3.2`! Thanks for flagging

**Comment by nemonemi** [comment]
Feel free to hit me up for those reviews ;)
